src/client/core/utils/dom.js

```diff
@@ -364,28 +367,81 @@ export function getElementDescription (el) {
     return res.join('');
 }
 
-export function getNextFocusableElement (element, reverse) {
-    const offset       = reverse ? -1 : 1;
-    let allFocusable = sortElementsByFocusingIndex(getFocusableElements(findDocument(element)));
+export function focusNextElement (element, reverse, skipRadioGroups) {
```

Comment:
Is domUtils the good place for `focusNextElement `? What do you think @AndreyBelym